include/lidartag/rectangle_estimator.hpp

```diff
@@ -15,6 +15,7 @@
 #ifndef RECTANGLE_ESTIMATOR_HPP
 #define RECTANGLE_ESTIMATOR_HPP
 
+//#include <Eigen/Dense> do not include this in lidartag...
```

Comment:
reason why?

Reply:
Actually, this is something that has been bothering me for the longest time, but I was not sure if we as Tier VI could spend time fixing this.

The authors at some point started using a custom version of eigen (hard copied into the repository), and not using it caused some compilation errors. However, since we use other libraries that use or interact with eigen (pcl, ros, opencv), there a quite some cases where inconsistencies occur, and just doing like some include cause compilation errors, or even worse data corruption.

Can I spend some time removing the custom version of eigen ?

Reply:
Hmm, okay so please create your open issue on github and our jira? For now, it's a little bit complex but there is no problem if our autoware build system is not affected.

Reply:
resolved?

Reply:
I created the ticket [(Link)](https://tier4.atlassian.net/browse/T4PB-17687), but I thought about doing it when all the current PRs have been merged since I may need to do several changes.